util/doxygen/Doxyfile

```diff
@@ -0,0 +1,200 @@
+#---------------------------------------------------------------------------
+# Project related configuration options
+#---------------------------------------------------------------------------
+
+PROJECT_NAME           = "Software APIs | OpenTitan"
```

Comment:
This gets placed to the right of the OT logo. Hence, the Doxygen page header reads

`OpenTitan* Software APIs | OpenTitan`

where `OpenTitan*` is the logo text. This feels a bit redundant, so maybe want to drop the `| OpenTitan` part...

Reply:
This was done so the HTML titles matched. I'll move the `| OpenTitan` part into the html header template for doxygen.

Reply:
Addressed.